### DISCUSSION TITLE
The domain works, but Vercel says invalid configurations

### DISCUSSION THREAD

**Discussion by ali-0315** [vercel/community#3169] [Feature Question] [answered]
### Question

Hi, I am using a cdn service providers which first redirects my domain to a page and checks whether the user is a robot or not and then it is transferred to the site, the domain works, but Vercel keeps sending emails warning of (invalid configurations), should I pay attention to it? is it possible to delete the domain?

**Comment by amyegan** [comment]
Hi @ali-0315. If you share the domain here, I can look into the reason for these warning emails.

If you're using Cloudflare, we have a setup guide that can help you make sure everything is configured appropriately: https://vercel.com/guides/using-cloudflare-with-vercel

**Comment by ali-0315** [comment]
ali0315.ir
I'm not using cloud flare

**Reply by amyegan** [reply, marked as answer]
It looks like you using a proxy service so [the `A` record](https://www.whatsmydns.net/#A/ali0315.ir) does not point to a Vercel IP and [the nameservers](https://www.whatsmydns.net/#NS/ali0315.ir) are not set to Vercel's nameservers. 

This is why you are receiving the configuration warning emails. You should also see a warning in your project's domain settings about `Invalid Configuration` with a instruction to add an `A` record with value `76.76.21.21`.

If the proxy service points to the default `*.vercel.app` domain for the project, then you might not need to include the domain in your project settings on Vercel.

**Reply by ali-0315** [reply]
Right, Thanks 🙏

**Reply by nikitaeverywhere** [reply]
@amyegan I'm using Cloudflare proxy for a domain to Vercel with Cloudflare Zero Trust (basically their SSO).

> then you might not need to include the domain in your project settings on Vercel

This unfortunately doesn't work.

- Whenever I add a domain to Vercel, everything works, but Vercel says "Invalid Configuration" and regularly spams me with emails every week.
   - FYI my Cloudflare CNAME is `cname.vercel-dns.com`.
- If I remove a domain like you suggested, then Vercel just says "there's no such app".
   - Even if I change CNAME to `myapp.vercel.app`, it doesn't work.

<details>
  <summary>Screenshots</summary>

<img width="692" alt="image" src="https://github.com/vercel/community/assets/4989256/83b17c6c-a3d6-4e4c-b7db-f4a0ec22e9d9">
<img width="885" alt="image" src="https://github.com/vercel/community/assets/4989256/b3b268f9-12a9-4f9a-97b9-da1d900d2be1">

</details>

[This is read](https://vercel.com/docs/integrations/cloudflare), doesn't work.

Any workaround?

**Comment by ProductOfAmerica** [comment]
@amyegan Any ideas? I don't have the nameservers enabled, because it works fine with another domain I own... without activating them. So I'm not sure why this domain isn't working...

### Vercel:
![chrome_J4UzhBMblv](https://github.com/vercel/community/assets/14712114/887b0901-ebd4-4264-83e6-89da493df918)

### AWS Route 53:
![chrome_OvuHIJrm8V](https://github.com/vercel/community/assets/14712114/a276745c-c820-4d7e-9c27-35c365fa3901)

### I even tried AWS's "Test record" feature and it's successful:
![chrome_YDLhJTaCqm](https://github.com/vercel/community/assets/14712114/f106b373-bd17-4738-992c-1cad7731dca1)

![image](https://github.com/vercel/community/assets/14712114/2913c80b-2428-4af3-952f-28cb29319d6f)

**Reply by hadnet** [reply]
I got this same error with another registrar. Did you find the solution? Thanks in advance, man.
